xyz

```diff
@@ -201,11 +219,6 @@ next_version=$(inc "${options[prerelease-label]}" \
 message="${options[message]//X.Y.Z/$next_version}"
 tag="${options[tag]//X.Y.Z/$next_version}"
 
-if type tput &>/dev/null ; then
-  bold=$(tput bold)
-  reset=$(tput sgr0)
-fi
-
```

Comment:
I moved this paragraph so I could use `bold` and `reset` in `check_bash_version`.